libkodimote/songs.cpp

```diff
@@ -210,10 +210,7 @@ void Songs::playItem(int row)
     } else {
         pItem.setAlbumId(m_albumId);
     }
-    Kodi::instance()->audioPlayer()->playlist()->clear();
-    Kodi::instance()->audioPlayer()->playlist()->addItems(pItem);
-//    Kodi::instance()->audioPlayer()->playlist()->playItem(row);
-    Kodi::instance()->audioPlayer()->playItem(row);
+    Kodi::instance()->audioPlayer()->open(pItem, resume);
```

Comment:
Fixed in new commit.

Added an `open` overload to audioPlaylist which also accepts the position. The current `open` uses the overload to start from `0`, and songs can start from the requested position.